tests/bluetooth/controller/ctrl_phy_update/src/main.c

```diff
@@ -294,6 +318,7 @@ ZTEST(phy_central, test_phy_update_central_loc_unsupp_feat)
 
 	/* Connect */
 	ull_cp_state_set(&conn, ULL_CP_CONNECTED);
+	ut_rx_q_is_empty();
```

Comment:
Have you run into a situation where this is not always true at the start of a test since you added this?

Reply:
I suspected, but didn't - I'll remove